spec/features/admin/sign_in_as_admin_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+
+describe 'signing as an admin user' do
+  let!(:user) { create(:user, :confirmed, admin: true) }
+
+  context 'when visiting the home page' do
+    before do
+      5.times { create(:organisation) }
```

Comment:
I don't think you need to test it with 5 different organisations. You get the same test coverage out of one.